spec/tree_spec.rb

```diff
@@ -1,14 +1,80 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
-  it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+describe AppleTree do
+  context 'when AppleTree exists' do
+    subject(:tree) { AppleTree.new }
+    it 'should be a Class' do
+      expect(described_class.is_a? Class).to be true
+    end
+
+    it 'should be a newborn tree' do
+      expect(tree.age).to eq 0
+    end
+
+    it 'should have no apples' do
+      expect(tree.any_apples?).to eq false
+    end
```

Comment:
For these two tests: if it's part of the required setup for these guys that this be a brand-new tree, then the setup should go with the test.  The options are either to do the necessary setup inside the `it` block itself, or else to use a `context`/`describe` block.